app/src/test/java/com/team7/cmput301/android/theirisproject/ProblemTest.java

```diff
@@ -10,19 +11,27 @@
 import org.junit.Test;
 
 import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
 
 public class ProblemTest {
 
-    private String title = "Zombification";
-    private String description = "I think I'm slowly turning into a zombie.";
-    private String userid = "0";
-    private RecordList records = new RecordList();
-    private List<BodyPhoto> body_photos = new ArrayList<>();
+    private String title;
+    private String description;
+    private String userid;
+    private RecordList records;
+    private List<BodyPhoto> body_photos;
+
+    private String _id;
+    private String problemId;
+    private String author;
+    private String role;
```

Comment:
"Role" should be migrated to type UserType, this might force you to have to make sure it doesn't cause problems anywhere else though

Reply:
After typing in User.UserType once make sure to alt+enter and add it as a static import so you won't have to keep typing User.UserType